curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<head>
+<meta charset = "utf-8">
+<title>OpenCV.js</title>
+</head>
+<body>
+<h2>OpenCV.js</h2>
+<p id = "status">OpenCV.js is loading...</p>
+<div>
+    <div class = "inputoutput">
+            <div class = "caption">
+                Run example:  <input type = "button" id = "myButton" onclick = "draw()" value = "Run" disabled = true />
+            </div>
+    </div>
+        <div class = "inputoutput">
+            <canvas id = "canvasOutput" >
+            </canvas>
+                <div class = "caption">Result
+                </div>
+        </div>
+</div>
+
+<script type = "text/javascript">
+
+//Source:
+//https://docs.opencv.org/2.4/modules/core/doc/drawing_functions.html
+
+function draw() {
+    let myButton = document.getElementById("myButton");
+    let mat = new cv.Mat(300, 300, cv.CV_8UC3, new cv.Scalar(255, 255, 255, 255));
+
+    cv.ellipse(mat, new cv.Point(220, 170), new cv.Size(50, 50), 300.0, 0.0, 300.0, new cv.Scalar(0, 0, 255, 255), cv.FILLED);
+    cv.ellipse(mat, new cv.Point(80, 170), new cv.Size(50, 50), 0.0, 0.0, 300.0, new cv.Scalar(0, 255, 0, 255), cv.FILLED);
+    cv.ellipse(mat, new cv.Point(150, 60), new cv.Size(50, 50), 120.0, 0.0, 300.0, new cv.Scalar(255, 0, 0, 255), cv.FILLED);
```

Comment:
Please add functions such
```js
function drawEllipse(img, center, radius, startAngle, endAngle, color)
```
In code we'll call it with right arguments - user should implement just method.